packages/phrases/src/locales/en/translation/admin-console/connector-details.ts

```diff
@@ -9,13 +9,14 @@ const connector_details = {
   save_error_empty_config: 'Please enter config',
   send: 'Send',
   send_error_invalid_format: 'Invalid input',
-  edit_config_label: 'Enter your json here',
+  edit_config_label: 'Enter your JSON here',
   test_email_sender: 'Test your email connector',
   test_sms_sender: 'Test your SMS connector',
   test_email_placeholder: 'Enter a test email address',
   test_sms_placeholder: 'Enter a test phone number',
-  test_message_sent: 'Test message sent!',
-  test_sender_description: 'You will receive a message if your json is rightly configured',
+  test_message_sent: 'Test message sent',
+  test_sender_description:
+    'Logto uses the "Generic" template for testing. You will receive a message if your JSON is rightly configured.',
```

Comment:
better to use: `You will receive a message if your connector is rightly configured.` since we are going to use form view.